### PR TITLE
Update CONTRIBUTING and README

### PR DESCRIPTION
**Pull request by boasvdp**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ From time to time, the owners of this repo will merge the dev branch with the ma
 Please see [Read Git Flow](https://leanpub.com/git-flow/read#leanpub-auto-git-flow-example) for more information,
 or also please see this depiction for our workflow: https://leanpub.com/site_images/git-flow/git-workflow-release-cycle-3release.png
 
+**Important:** in the basic templates, triggering tests on push events is switched off by default (e.g. lines 8 and 9 in `prokka.yaml`).
```
Comment (lskatz) [inline CONTRIBUTING.md]: Please remove this statement as we have learned more about when minutes are counted against us.

File: .github/workflows/quast.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 name: quast
 
 on:
-  push:
-    branches: [ main, dev ]
+#  push:
```
Comment (lskatz) [inline .github/workflows/quast.yml]: Please revert

File: .github/workflows/prokka.yml [hunk 3]
```diff
@@ -5,8 +5,8 @@ name: prokka
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the main branch
 on:
-  push:
-    branches: [ main, dev ]
+#  push:
```
Comment (lskatz) [inline .github/workflows/prokka.yml]: Please revert

File: .github/workflows/CSIS.yml [hunk 4]
```diff
@@ -5,8 +5,8 @@ name: CSIS
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the main branch
 on:
-  push:
-    branches: [ main, dev ]
+#  push:
```
Comment (lskatz) [inline .github/workflows/CSIS.yml]: Please revert

**Review by lskatz — CHANGES_REQUESTED** [review]
We didn't know this for sure until today but our minutes seem limitless.  Therefore please revert some of these changes as indicated.

**Comment by boasvdp** [comment]
The changes around test minute limits have been reverted, I've kept the rationale for Actions in README

**Pull request merged**